PKHUD/PKHUD.swift

```diff
@@ -92,16 +102,29 @@ open class PKHUD: NSObject {
             container.frame.origin = CGPoint.zero
             container.frame.size = view.frame.size
             container.autoresizingMask = [ .flexibleHeight, .flexibleWidth ]
+            container.isHidden = true
         }
-        container.showFrameView()
         if dimsBackground {
             container.showBackground(animated: true)
         }
 
-        startAnimatingContentView()
+        finished = false
+
+        // If the grace time is set, postpone the HUD display
+        if graceTime > 0.0 {
+            let timer = Timer(timeInterval: graceTime, target: self, selector: #selector(PKHUD.handleGraceTimer(_:)), userInfo: nil, repeats: false)
```

Comment:
You copy  the following:
```swift
            container.showFrameView()
            startAnimatingContentView()
```
in the else statement and in `PKHUD.handleGraceTimer(_:) `. Please use a function